src/Servers/HttpSys/src/NativeInterop/RequestQueue.cs

```diff
@@ -143,6 +174,9 @@ public void Dispose()
         }
 
         _disposed = true;
+
+        PInvoke.HttpCloseRequestQueue(Handle);
```

Comment:
Seems like a reasonable change but unrelated to this PR. Drive-by bug fix or is it related somehow?

Reply:
Drive-by, was just reading the docs for HttpCreateRequestQueue to make sure I was doing the correct thing with the security attribute and noticed it said you should close the queue handle with HttpCloseRequestQueue.